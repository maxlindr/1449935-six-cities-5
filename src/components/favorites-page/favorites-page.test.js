import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FavoritesPage from './favorites-page';
import offers from '../../../__mocks__/offer-mocks';

const renderer = new ShallowRenderer();

test(`FavoritesPage должен корректно отображаться`, () => {
  renderer.render(
      <FavoritesPage offers={offers} />
  );

  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
