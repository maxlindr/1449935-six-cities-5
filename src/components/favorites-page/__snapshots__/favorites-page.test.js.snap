// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesPage должен корректно отображаться 1`] = `
<div
  className="page"
>
  <Memo(Connect(PageHeader)) />
  <main
    className="page__main page__main--favorites"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites"
      >
        <h1
          className="favorites__title"
        >
          Saved listing
        </h1>
        <ul
          className="favorites__list"
        >
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Cologne
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 3,
                    "description": "Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.",
                    "favorite": false,
                    "features": Array [
                      "Laptop friendly workspace",
                      "Washer",
                      "Towels",
                      "Fridge",
                      "Air conditioning",
                      "Baby seat",
                      "Breakfast",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "1",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          50.938361,
                          6.959974,
                        ],
                        "name": "Cologne",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        50.957361,
                        6.9509739999999995,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 4,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                    ],
                    "premium": false,
                    "price": 493,
                    "rating": 3.8,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                    "title": "The house among olive ",
                    "type": "house",
                  }
                }
                type="favorite"
              />
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 3,
                    "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
                    "favorite": false,
                    "features": Array [
                      "Towels",
                      "Laptop friendly workspace",
                      "Washer",
                      "Baby seat",
                      "Breakfast",
                      "Air conditioning",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "9",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          50.938361,
                          6.959974,
                        ],
                        "name": "Cologne",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        50.930361,
                        6.937974,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 9,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                    ],
                    "premium": false,
                    "price": 308,
                    "rating": 4.9,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                    "title": "Canal View Prinsengracht",
                    "type": "hotel",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Brussels
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 3,
                    "description": "This is a place for dreamers to reset, reflect, and create. Designed with a \`slow\` pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.",
                    "favorite": false,
                    "features": Array [
                      "Towels",
                      "Baby seat",
                      "Laptop friendly workspace",
                      "Dishwasher",
                      "Washing machine",
                      "Air conditioning",
                      "Washer",
                      "Coffee machine",
                      "Breakfast",
                      "Fridge",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "2",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          50.846557,
                          4.351697,
                        ],
                        "name": "Brussels",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        50.839557,
                        4.346697,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 8,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                    ],
                    "premium": false,
                    "price": 323,
                    "rating": 4.3,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                    "title": "Amazing and Extremely Central Flat",
                    "type": "apartment",
                  }
                }
                type="favorite"
              />
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 4,
                    "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
                    "favorite": false,
                    "features": Array [
                      "Washer",
                      "Laptop friendly workspace",
                      "Breakfast",
                      "Air conditioning",
                      "Baby seat",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "3",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          50.846557,
                          4.351697,
                        ],
                        "name": "Brussels",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        50.865556999999995,
                        4.371696999999999,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 7,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                    ],
                    "premium": false,
                    "price": 218,
                    "rating": 4.6,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                    "title": "The house among olive ",
                    "type": "apartment",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Amsterdam
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 3,
                    "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
                    "favorite": false,
                    "features": Array [
                      "Dishwasher",
                      "Air conditioning",
                      "Laptop friendly workspace",
                      "Washer",
                      "Baby seat",
                      "Towels",
                      "Breakfast",
                      "Coffee machine",
                      "Fridge",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "4",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          52.37454,
                          4.897976,
                        ],
                        "name": "Amsterdam",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        52.385540000000006,
                        4.902976,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 7,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                    ],
                    "premium": false,
                    "price": 440,
                    "rating": 4.6,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                    "title": "Perfectly located Castro",
                    "type": "house",
                  }
                }
                type="favorite"
              />
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 1,
                    "description": "A new spacious villa, one floor. All commodities, jacuzzi and beautiful scenery. Ideal for families or friends.",
                    "favorite": false,
                    "features": Array [
                      "Laptop friendly workspace",
                      "Washer",
                      "Breakfast",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "5",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          52.37454,
                          4.897976,
                        ],
                        "name": "Amsterdam",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        52.388540000000006,
                        4.899976,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 3,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                    ],
                    "premium": false,
                    "price": 134,
                    "rating": 2.2,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                    "title": "The house among olive ",
                    "type": "room",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Paris
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 5,
                    "description": "Peaceful studio in the most wanted area in town. Quiet house Near of everything. Completely renovated. Lovely neighbourhood, lot of trendy shops, restaurants and bars in a walking distance.",
                    "favorite": false,
                    "features": Array [
                      "Dishwasher",
                      "Fridge",
                      "Towels",
                      "Washer",
                      "Breakfast",
                      "Baby seat",
                      "Laptop friendly workspace",
                      "Air conditioning",
                      "Coffee machine",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "6",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          48.85661,
                          2.351499,
                        ],
                        "name": "Paris",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        48.877610000000004,
                        2.333499,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 8,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                    ],
                    "premium": false,
                    "price": 226,
                    "rating": 3,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                    "title": "The house among olive ",
                    "type": "apartment",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Hamburg
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 2,
                    "description": "Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.",
                    "favorite": false,
                    "features": Array [
                      "Washer",
                      "Fridge",
                      "Air conditioning",
                      "Baby seat",
                      "Breakfast",
                      "Towels",
                      "Laptop friendly workspace",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "7",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          53.550341,
                          10.000654,
                        ],
                        "name": "Hamburg",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        53.538341,
                        9.976654000000002,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 3,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                    ],
                    "premium": false,
                    "price": 277,
                    "rating": 2.3,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                    "title": "Loft Studio in the Central Area",
                    "type": "hotel",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
          <li
            className="favorites__locations-items"
          >
            <div
              className="favorites__locations locations locations--current"
            >
              <div
                className="locations__item"
              >
                <a
                  className="locations__item-link"
                  href="#"
                >
                  <span>
                    Dusseldorf
                  </span>
                </a>
              </div>
            </div>
            <div
              className="favorites__places"
            >
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 1,
                    "description": "Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.",
                    "favorite": false,
                    "features": Array [
                      "Baby seat",
                      "Fridge",
                      "Towels",
                      "Air conditioning",
                      "Breakfast",
                      "Washer",
                      "Laptop friendly workspace",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "8",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          51.225402,
                          6.776314,
                        ],
                        "name": "Dusseldorf",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        51.204402,
                        6.7773140000000005,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 1,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
                    ],
                    "premium": false,
                    "price": 129,
                    "rating": 2.3,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg",
                    "title": "Amazing and Extremely Central Flat",
                    "type": "room",
                  }
                }
                type="favorite"
              />
              <Memo(OfferCard)
                offer={
                  Object {
                    "bedrooms": 1,
                    "description": "Design interior in most sympathetic area! Complitely renovated, well-equipped, cosy studio in idyllic, over 100 years old wooden house. Calm street, fast connection to center and airport.",
                    "favorite": true,
                    "features": Array [
                      "Breakfast",
                      "Laptop friendly workspace",
                      "Air conditioning",
                      "Washer",
                    ],
                    "host": Object {
                      "avatar": "img/avatar-angelina.jpg",
                      "id": 25,
                      "name": "Angelina",
                      "super": true,
                    },
                    "id": "10",
                    "location": Object {
                      "city": Object {
                        "coordinates": Array [
                          51.225402,
                          6.776314,
                        ],
                        "name": "Dusseldorf",
                        "zoom": 13,
                      },
                      "coordinates": Array [
                        51.211402,
                        6.756314000000001,
                      ],
                      "zoom": 16,
                    },
                    "maxAdults": 2,
                    "photos": Array [
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg",
                      "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg",
                    ],
                    "premium": false,
                    "price": 125,
                    "rating": 4.2,
                    "thumbnail": "https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg",
                    "title": "Penthouse, 4-5 rooms + 5 balconies",
                    "type": "apartment",
                  }
                }
                type="favorite"
              />
            </div>
          </li>
        </ul>
      </section>
    </div>
  </main>
  <PageFooter
    container={true}
  />
</div>
`;
